Prometheus, Loki: Update run queries button (#50005)
grafana/grafana: docs/sources/datasources/loki.md

Loki query editor is separated into 3 distinct modes that you can switch between. See docs for each section below.

> At the top of the editor there is `Run queries` button that will run the queries and `Explain | Builder | Code` tabs to switch between the editor modes. If the query editor is in Builder mode there are additional elements explained in the Builder section.
> 
> > **Note:** Run queries button is not present in Explore. To run Loki queries, use Explore's `Run query` button.

Reviewer: ```suggestion
At the top of the editor, select `Run queries` to run a query. Select `Explain | Builder | Code` tabs to switch between the editor modes. If the query editor is in Builder mode, there are additional elements explained in the Builder section.

> **Note:** In Explore, `Run queries` is called `Run query`. To run Loki queries, select `Run query`.
```

Author: @brendamuir the changes looks good, but this below is not correct. We don't have that button in query editor in Explore and instead we have 1 button at the top of Explore.

```
In Explore, `Run queries` is called `Run query`. To run Loki queries, select `Run query`.
```

This is editor in dashboard:

<img width="933" alt="image" src="https://user-images.githubusercontent.com/30407135/171584461-ca5f7f42-e9d0-42c9-b59e-7be527129c28.png">

This is editor in explore. No Run queries button, but there is Run query button on top ox Explore.
<img width="1446" alt="image" src="https://user-images.githubusercontent.com/30407135/171584576-33962393-45ea-4f1f-b4ec-9e3b1575c857.png">

Reviewer: **Note:** In Explore, to run Loki queries, select `Run query`. does that make more sense?

Author: Yes!
